marimo/_server/sessions.py

```diff
@@ -116,12 +116,14 @@ def __init__(
         mode: SessionMode,
         configs: dict[CellId_t, CellConfig],
         app_metadata: AppMetadata,
+        package_manager: str,
     ) -> None:
         self.kernel_task: Optional[threading.Thread] | Optional[mp.Process]
         self.queue_manager = queue_manager
         self.mode = mode
         self.configs = configs
         self.app_metadata = app_metadata
+        self.package_manager = package_manager
```

Comment:
The kernel doesn't have access to the user config, so I plumbed down what we needed from it